pom.xml

```diff
@@ -27,7 +27,7 @@
         <alfresco.outlook.version>2.9.0</alfresco.outlook.version>
         <alfresco.transformation-engine.version>2.3.1</alfresco.transformation-engine.version>
         <alfresco.desktop-sync.version>3.11.3</alfresco.desktop-sync.version>
-        <alfresco.ais.version>1.4.2</alfresco.ais.version>
+        <alfresco.ais.version>3.1.3</alfresco.ais.version>
```

Comment:
This won't work. For 7.2.N we've decided to use old AIS AMP + latest ATS/AIS components (T-Engines  & T-Router)